test/extensions/clusters/aggregate/cluster_test.cc

```diff
@@ -131,6 +134,24 @@ class AggregateClusterTest : public Event::TestUsingSimulatedTime, public testin
     lb_ = lb_factory_->create(lb_params_);
   }
 
+  Stats::Gauge& getCircuitBreakersStatByPriority(std::string priority, std::string stat) {
+    std::string stat_name_ = "circuit_breakers." + priority + "." + stat;
```

Comment:
Usually the trailing underscore (`_`) is only for private class member variables.
This is a local variable, so it should not have a trailing underscore.

```suggestion
    std::string stat_name = "circuit_breakers." + priority + "." + stat;
```
(and similar on the following line)

Reply:
@SaadiaELF this PR https://github.com/bazmurphy/envoy/pull/23 fixes that^